ocs_ci/ocs/must_gather/must_gather.py

```
@@ -188,6 +190,39 @@ def verify_noobaa_diagnostics(self):
                 logger.error("noobaa_diagnostics.tar.gz does not exist")
                 self.files_not_exist.append("noobaa_diagnostics.tar.gz")
 
+    def get_must_gather_pod(self):
+        """
+
+        :return:
+        """
+        self.mg_pod = get_pod_name_by_pattern(
+            pattern=must_gather_pod_label, namespace=OPENSHIFT_STORAGE_NAMESPACE
+        )
+        if len(self.mg_pod) > 0:
+            return True
+        else:
+            return False
```

Should we throw exception if the must-gather pod is not found?